### ISSUE TITLE
OpenIddict: Override UserInfoController

### ISSUE THREAD

**Issue by merijng** [abpframework/abp#15562]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

I am trying to override the [OpenIddict UserInfo endpoint](https://github.com/abpframework/abp/blob/581e6cc291e886143706b2393c221f2ed0d7ce71/modules/openiddict/src/Volo.Abp.OpenIddict.AspNetCore/Volo/Abp/OpenIddict/Controllers/UserInfoController.cs), so I can add one extra claim.

However, it returns a `IActionResult`. Therefor it is complex to add an extra claim to the dictionary, after calling the base method. It is not possible to do the following:

```CSHARP
var claims = base.UserInfo();
claims.Add(...);
return claims
```

### Describe the solution you'd like

Isn't it better to seperate the code, for example by creating a protected method within the controller? This method should return a dictionary of clients.

### Additional context

- We don't want to copy the whole code, because changes with updating ABP.io are potentially not seen
- I think I am not the only person that want's to edit the UserInfo endpoint, by just adding one or two claims.

**Comment by merijng** [comment]
See pull request: #15565

**Comment by hikalkan** [comment]
Closed by #15565
